docs/UserGuide.md

```diff
@@ -11,45 +11,40 @@ title: User Guide
-
 * Items in square brackets are optional.<br>
   e.g `n/NAME [c/CLASS_NAME]` can be used as `n/John Doe c/CS1231S` or as `n/John Doe`.
-
 * ​Items with `...` after them can be used multiple times including zero times.<br>
   e.g. `[c/CLASS_NAME]...` can be used as ` ` (i.e. 0 times), `c/CS1101S`, `c/CS2030 c/ST2334` etc.
-
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
-
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
-</div>
+" %}
 
-| Command    | Format                                                                       |
-|------------|------------------------------------------------------------------------------|
-| `help`     | `help`                                                                       |
-| `add`      | `add n/NAME [p/PHONE_NUMBER][e/EMAIL] [a/ADDRESS][c/CLASS_NAME...]`          |
+| Command    | Format                                   |
+| ---------- | ---------------------------------------- |
+| `help`     | `help`                                   |
+| `add`      | `add n/NAME [p/PHONE_NUMBER][e/EMAIL] [a/ADDRESS][c/CLASS_NAME...]` |
 | `edit`     | `edit INDEX [n/NAME][p/PHONE_NUMBER] [e/EMAIL][a/ADDRESS] [c/CLASS_NAME...]` |
-| `delete`   | `delete INDEX`                                                               |
-| `find`     | `find KEYWORD [MORE_KEYWORDS...]`                                            |
-| `list`     | `list`                                                                       |
-| `addc`     | `addc c/CLASS_NAME`                                                          |
-| `deletec`  | `deletec c/CLASS_NAME`                                                       |
-| `assign`   | `assign INDEX... c/CLASS_NAME`                                               |
-| `unassign` | `unassign INDEX... c/CLASS_NAME`                                             |
-| `listc`    | `listc`                                                                      |
-| `exit`     | `exit`                                                                       |
-| `focus`    | `focus c/CLASS_NAME`                                                         |
+| `delete`   | `delete INDEX`                           |
+| `find`     | `find KEYWORD [MORE_KEYWORDS...]`        |
+| `list`     | `list`                                   |
+| `addc`     | `addc c/CLASS_NAME`                      |
+| `deletec`  | `deletec c/CLASS_NAME`                   |
```

Comment:
Both addc and deletec should contain `...` behind `c/CLASS_NAME` now that adding and deleting multiple classes is possible

Reply:
Fixed!